app/scripts/modules/google/serverGroup/configure/wizard/customInstance/customInstanceBuilder.gce.service.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.serverGroup.customInstanceBuilder.gce.service', [
+  require('../../../../../core/utils/lodash.js')
+])
+  .factory('gceCustomInstanceBuilderService', function(_) {
+    /*
+    * Zones that support Haswell and Ivy Bridge processors can support custom machine types up to 32 vCPUs.
```

Comment:
Please add a link to this comment to keep it in sync with the related list in clouddriver (should link to the class file). Please also add a link to the clouddriver list pointing back here.